### PR TITLE
Added post about cerner_kafka chef cookbook

### PR DESCRIPTION
**Pull request by bbaugher**

### CODE REVIEW

File: source/_posts/2017-02-13-automation-deployment-with-apache-kafka.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+date: 2017-02-13
+tags: [engineering, kafka]
+---
+
+Its likely not a surprise that Cerner would use Apache Kafka as we have used
+a number of related technologies like [Apache Hadoop](http://engineering.cerner.com/2013/02/composable-mapreduce-with-hadoop-and-crunch/)
+along with its Map/Reduce, HDFS and even [Apache HBase](http://engineering.cerner.com/2013/02/near-real-time-processing-over-hadoop-and-hbase/).
+Our team first started using [Apache Kafka](http://kafka.apache.org/) in 2014
+when Kafka 0.8 first came out. Since then we've expanded to using Kafka for a
+number of different use cases and it has become a core piece of Cerner's infrastructure.
```
Comment (mkwhitacre) [inline source/_posts/2017-02-13-automation-deployment-with-apache-kafka.markdown]: Optional blog/videos we can link to for expanding on our use cases 
http://blog.cloudera.com/blog/2014/11/how-cerner-uses-cdh-with-apache-kafka/
https://www.confluent.io/kafka-summit-2016-users-ingesting-complex-healthcare-data-with-apache-kafka

Comment (bbaugher) [inline source/_posts/2017-02-13-automation-deployment-with-apache-kafka.markdown]: Added, https://github.com/cerner/cerner.github.io/pull/58/commits/a3696e90d18dca71e4fdaac9671966ecf63907fa

**Comment by mkwhitacre** [comment]
+1

File: source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Automated Deployment with Apache Kafka"
+author: Bryan Baugher
+date: 2017-02-13
+tags: [engineering, kafka]
+---
+
+Its likely not a surprise that Cerner would use Apache Kafka as we have used
```
Comment (cchesser) [inline source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown]: "Its" -> "It's"

Comment (cchesser) [inline source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown]: I would move your hyperlink to Kafka to its first reference.

File: source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+a number of related technologies like [Apache Hadoop](http://engineering.cerner.com/2013/02/composable-mapreduce-with-hadoop-and-crunch/)
+along with its Map/Reduce, HDFS and even [Apache HBase](http://engineering.cerner.com/2013/02/near-real-time-processing-over-hadoop-and-hbase/).
+Our team first started using [Apache Kafka](http://kafka.apache.org/) in 2014
+when Kafka 0.8 first came out. Since then we've expanded to using Kafka for a
+number of different use cases
+([1](http://blog.cloudera.com/blog/2014/11/how-cerner-uses-cdh-with-apache-kafka/), [2](https://www.confluent.io/kafka-summit-2016-users-ingesting-complex-healthcare-data-with-apache-kafka))
+and it has become a core piece of Cerner's infrastructure.
+
+Just like the [applications we create](http://engineering.cerner.com/blog/deploying-web-services-with-apache-tomcat-and-chef/)
```
Comment (cchesser) [inline source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown]: Add comma:

> Just like the applications we create, we also needed ...

File: source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+we also needed to automate the deployment of Kafka to handle the ever growing
+amount of operations work and to ensure a high level of consistency with our
+environments. Chef was an obvious choice for us since its been our deployment tool of
```
Comment (cchesser) [inline source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown]: Minor: Add a hyperlink on the reference to Chef.

File: source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+choice for the last few years. We quickly put together a
+[cookbook](https://github.com/cerner/cerner_kafka) to help us automate the
+deployment of the many Kafka clusters here at Cerner. We have
+[continued to support and use this cookbook](https://github.com/cerner/cerner_kafka/blob/master/CHANGELOG.md)
+after open sourcing it by upgrading it to handle newer versions of Kafka (0.9 and 0.10),
```
Comment (cchesser) [inline source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown]: "open sourcing" -> "open-sourcing"

Comment (cchesser) [inline source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown]: Minor grammar changes / including link to fixed issues:

> by: upgrading it to handle newer versions of Kafka (0.9, 0.10), handle Kerberos authentication, and [many other improvements](https://github.com/cerner/cerner_kafka/issues?q=is%3Aissue%20is%3Aclosed%20).

File: source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+handle Kerberos authentication and many other improvements.
+
+If you use Apache Kafka feel free to [try it out](https://github.com/cerner/cerner_kafka)
```
Comment (cchesser) [inline source/_posts/2017-02-13-automated-deployment-with-apache-kafka.markdown]: Add comma:
> If you use Apache Kafka, feel free ...

**Comment by bbaugher** [comment]
Hopefully I fixed everything

**Comment by cchesser** [comment]
👍

**Comment by anelson425** [comment]
I would update the date for 2/16 and we can get this published tomorrow.

**Comment by anelson425** [comment]
+1

**Comment by bbaugher** [comment]
Updated to 2-16

**Comment by cchesser** [comment]
Merged: 	89ef4c7
Deployed site: http://engineering.cerner.com/blog/automated-deployment-with-apache-kafka/

**Pull request closed without merging**